.ci/.e2e-platforms.yaml

```
@@ -40,3 +40,13 @@ PLATFORMS:
     image: "ami-0d90bed76900e679a"
     instance_type: "t3.xlarge"
     username: admin
+  debian_11_amd64:
+    description: "Debian 11 AMD64"
+    image: "ami-0c6acff4e66eba1f1"
+    instance_type: "t3.xlarge"
+    username: admin
+  ubuntu_22.04:
```

I'd say `ubuntu_22.04` is not a good YAML key:
```suggestion
  ubuntu_22_04_amd:
```

---

made the changes